rasa/core/domain.py

```diff
@@ -191,11 +191,29 @@ def merge_lists(l1, l2):
 
         return self.__class__.from_dict(combined)
 
+    @classmethod
+    def _check_spaces_between_utters(cls, yaml):
+
+        domain_lines = yaml.split('\n')
+        for line in domain_lines:
+            line_s = line.strip().split('_')
+            if len(line_s) >= 2 and line_s[0] == 'utter':
+                index = domain_lines.index(line) - 1
+
+                previous_line = domain_lines[index]
+
+                if (previous_line != '\n' and
+                        previous_line != 'templates:\n'):
+                    logger.warning("There should be a space between lines"
```

Comment:
```suggestion
                    logger.warning("There should be a new line between lines"
```

So what happens in your experience with this error, the bot doesn't train or doesn't respond with the template when it calls the utterance?  I was able to do both

Reply:
When there isn't a space between the 2 utters the 2nd one can't be accessed, and the bot never uses it, even when there is a story for it. It trains correctly and doesn't show any mistakes, just while performing it doesn't use the utterance.

Reply:
Do you mean formatted like this?
```
  utter_cheer_up:
  - text: "Here is something to cheer you up:"
    image: "https://i.imgur.com/nGF1K8f.jpg"
  utter_did_that_help:
  - text: "Did that help you?"
  utter_happy:
  - text: "Great carry on!"
```
Or like so?
```
  utter_greet:
  - text: "Hey! How are you?"
  - text: "hiya!!"
  - text: "oh hey"
```
Either way, in local testing mine works fine -- can't reproduce the issue you're talking about

Reply:
Hi, I was having this problem with another project, but it was an older version of rasa, I tested again and it seems like this problem is already solved. I'll remove this part of the code, thanks for the help. :)